.golangci.yml

```diff
@@ -7,13 +7,17 @@ run:
 linters:
   disable-all: true
   enable:
+    - asasalint
     - asciicheck
     - bodyclose
     - containedctx
+    - decorder
```

Comment:
You can work around this by adding "decorder" to the .codespellignore file.

Reply:
Done.